### DISCUSSION TITLE
Can Siri switch the input?

### DISCUSSION THREAD

**Discussion by PudMuddle** [ACDR/homebridge-yamaha-avr#59] [Q&A]
Hi... Great plug-in!  It works great for me in the Home app interface.  But is there a phrase that will allow Siri to change the input source?  The ones I've tried thouroghly confuse her.   :-)

**Comment by Kaici433** [comment]
You can create a shortcut, that switches to a static input, as workaround. I did not find a way to pass the input as parameter.

**Comment by Kienz** [comment]
You can create a Scene in HomeKit with the specific input and tell Siri to activate this scene.
